src/utils/utils.ts

```diff
@@ -551,7 +551,15 @@ export function getSourceFromLocations(
 export function runStarkNetClassHash(filePath: string): string {
   const warpVenvPrefix = `PATH=${path.resolve(__dirname, '..', '..', 'warp_venv', 'bin')}:$PATH`;
 
-  const classHash = execSync(`${warpVenvPrefix} starknet-class-hash ${filePath}`).toString().trim();
+  let classHash: string;
+  try {
+    classHash = execSync(`${warpVenvPrefix} starknet-class-hash ${filePath}`).toString().trim();
+  } catch (error) {
+    throw new TranspileFailedError(
+      `starknet-class-hash command do not work as expected.\n Please make sure you are using the required version of starknet by executing 'pip install -r requirements.txt'`,
```

Comment:
`pip install -r requirement` is for us but for the user it would be to run `warp install`

Reply:
look at the warp install command to see how its used